app/assets/stylesheets/helpers/_contents-with-body.scss

```diff
@@ -0,0 +1,28 @@
+.contents-list-link-container {
+  margin: 0 auto;
+
+  @include media(tablet) {
+    max-width: 990px;
+  }
+
+  .app-c-back-to-top {
+    margin-left: 0;
+  }
+}
+
+.contents-list-link-wrapper.govuk-sticky-element--stuck-to-window {
+  background-color: $panel-colour;
+  display: inline;
```

Comment:
Are `display: inline` and `float: left` needed? I tried toggling them off and it didn't seem to make a difference.

Reply:
@vanitabarrett you're right - they aren't necessary - will remove.